src/main/java/htsjdk/samtools/fastq/FastqReader.java

```diff
@@ -41,6 +41,22 @@
  * directly.  It is provided so that this class can be used in Java for-each loop.
  */
 public class FastqReader implements Iterator<FastqRecord>, Iterable<FastqRecord>, Closeable {
+    /** Enum of the types of lines we see in Fastq. */
+    protected enum LineType {
+        SequenceHeader("Sequence Header"),
+        SequenceLine("Sequence Line"),
+        QualityHeader("Quality Header"),
+        QualityLine("Quality Line");
+
+        private String printable;
+
+        LineType(String printable) {
+            this.printable = printable;
+        }
+
+        @Override public String toString() { return this.printable; }
```

Comment:
code-cov shows that this `toString` doesn't get called, at all. 

Can I get a small test from you?